dpxdt/tools/site_diff.py

```diff
@@ -293,6 +299,11 @@ def run(self,
             if FLAGS.cookies:
                 config_dict['cookies'] = json.loads(
                     open(FLAGS.cookies).read())
+            if http_username and http_password:
```

Comment:
May be worth splitting this into two if statements so we can support http_username being set but http_password not being set?